ospfd/ospf_vty.c

```diff
@@ -8322,8 +8324,15 @@ DEFUN (ip_ospf_hello_interval,
 
 		params = ospf_get_if_params(ifp, addr);
 		ospf_if_update_params(ifp, addr);
+		is_addr = true;
 	}
 
+	old_interval = params->v_hello;
+
+	/* Return, if same interval is configured. */
+	if (old_interval == seconds)
```

Comment:
nit: IMO `old_interval` is unnecessary if it's only used for this comparison

Reply:
Yes , here old_interval used only to compare with new value and it ensure to return if the same value configured so that it will avoid unnecessary  restart of hello timer and hello Pkt transmission.